troubleshooting.md

```diff
@@ -1217,3 +1217,9 @@ WARN[0000] Can't stat lower layer "/var/lib/containers/storage/overlay/l/7HS76F2
 It is the user responsibility to make sure images in an additional
 store are not deleted while being used by containers in another
 store.
+
+### 36) Syncing bugfixes for podman-remote or setups using Podman API
+
+While looking for fixes or upgrading your podman version for the required fixes users often make mistake of only upgrading `podman` client however if a setup is using `podman-remote` or uses a client which communicates with `podman` server on remote machine via API. It is always recommended to upgrade version of `podman` client and `podman` server running on the remote machine and make sure that `podman` client and `podman` server point to the same version for the best experience.
+
+Example: If a particular bug was fixed in `v4.1.0` then `podman-client` must have version `v4.1.0` as well `podman-server` must have version `v4.1.0` for the best experience.
```

Comment:
Needs to have a quick description, a symptom section, and then a solution section.  Perhaps

```
After upgrading Podman to a newer version an issue with the earlier version of Podman still presents itself while using podman-remote.

#### Symptom

While running podman remote commands with the most updated Podman, issues that were fixed in a prior version of Podman can arise either on the Podman client side or the Podman server side.

#### Solution

When upgrading Podman to a particular version for the required fixes, users often make the mistake of only upgrading the Podman client.  However, if a setup is using `podman-remote` or uses a client which communicates with the Podman server on a remote machine via the REST API. It is required to upgrade both the Podman client and the Podman server running on the remote machine.  Both the Podman client and server must be upgraded to the same version.

Example: If a particular bug was fixed in `v4.1.0` then The Podman client` must have version `v4.1.0` as well the Podman server must have version `v4.1.0`.
```
feel free to tweak.